### ISSUE TITLE
Add meta tags

### ISSUE THREAD

**Issue by SheepTester** [nbuilding/nbuilding.github.io#3]
It'd be cool if there was a favicon (a `favicon.ico` file at the root should suffice) and some meta tags to set a description and theme colour for a fancy embed. A thumbnail would be overkill, though

**Comment by chinmaym07** [comment]
Hey can I try this.!

**Comment by saumyasinghal747** [comment]
Thumbnail good

**Comment by SheepTester** [comment]
Sure

**Comment by mg27713** [comment]
"good first issue" Did you create that tag? I feel like you've given yourself that tag before on personal repos.

**Comment by SheepTester** [comment]
> "good first issue" Did you create that tag? I feel like you've given yourself that tag before on personal repos.

It's a default tag lol

**Comment by mg27713** [comment]
It seems like that tag is actually meant to be for people who aren't used to reviewing issues to find good issues to get started with.

**Comment by SheepTester** [comment]
> It seems like that tag is actually meant to be for people who aren't used to reviewing issues to find good issues to get started with.

Yeah, "good first issue" means new people can help resolve the issue

**Comment by mg27713** [comment]
You seemed to be putting it on every first issue you put on your own repos lol
